.github/workflows/bump.yml

```diff
@@ -56,19 +56,10 @@ jobs:
       - name: Bump version
         run: ${{ inputs.bump-command }}
 
-      - name: Get package.json version
-        run: echo VERSION=$(node -p 'require("./package.json").version') >> $GITHUB_ENV
-
       - name: Commit and push
         run: |
           git config --local user.email "deployooor@nexusmutual.io"
           git config --local user.name "Version bumpooor"
-          git add .
-          git commit -m "chore: release v${{ env.VERSION }} [skip ci]"
+          git add package.json package-lock.json
+          git commit -m "chore(release): v$(node -p 'require("./package.json").version') [skip ci]"
           git push origin ${{ inputs.ref }}
-
-      - name: Tag
-        if: inputs.ref == 'master'
-        run: |
-          git tag v${{ env.VERSION }}
-          git push origin --tags
```

Comment:
I feel like re-usuable workflows should be modular and focused on one thing.

When I first ran our workflow it was not immediately clear that the git tag happens inside here.

So for the visibility and explicitness I elected to move this git tag as a step in `ff-release-candidate-to-master.yml`